ci/azure/pipelines.yml

```diff
@@ -40,12 +40,15 @@ jobs:
   timeoutInMinutes: 360
 
   steps:
+  - powershell: |
+      (New-Object Net.WebClient).DownloadFile("https://github.com/msys2/msys2-installer/releases/download/2020-05-22/msys2-base-x86_64-20200522.sfx.exe", "sfx.exe")
```

Comment:
> 20200522

Why not 20200602?

Reply:
I just copied the one from here: https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml

Should be be using the latest one?

Reply:
> Should be be using the latest one?

I don't know. It's just my habit of using the latest software. :)

Reply:
updated